web/src/components/overlay/detail/SearchDetailDialog.tsx

```diff
@@ -773,8 +783,16 @@ function ObjectDetailsTab({
               </div>
             </div>
             <div className="text-sm">
-              {score}%{subLabelScore && ` (${subLabelScore}%)`}
+              {topScore}%{subLabelScore && ` (${subLabelScore}%)`}
+            </div>
+          </div>
+          <div className="flex flex-col gap-1.5">
```

Comment:
Probably want to check to see if the object has a snapshot and hide this if not

Reply:
oh yeah I literally thought that... and then forgot to put it in